packages/cli/test-node/RocketCli.preset.test.js

```diff
@@ -19,7 +19,7 @@ describe('RocketCli preset', () => {
     }
   });
 
-  it('offers a default layout (with head, header, content, footer, bottom) and raw layout', async () => {
+  it.only('offers a default layout (with head, header, content, footer, bottom) and raw layout', async () => {
```

Comment:
Did you add the `.only` by mistake?

Reply:
oh no🤦‍♂️

Reply:
thanks for checking 🤗. will fix it in a different PR